3-extra/1-radio-stations.js

```diff
@@ -13,11 +13,26 @@
  * - Should return this array to use in other functions
  */
 
+
 // `getAllFrequencies` goes here
 
+ function getAllFrequencies(){
+  let startFrequency = 87
+  let arrayOfFrequencies = []
+  while(startFrequency <= 108){
```

Comment:
Good work on this function it looks good. 

I have a couple of minor comments to improve the code:

Try to avoid hardcoded values like this '108'. Follow the example of the `let startFrequency = 87`.

The other comment that I have is regarding the while loop. While loops are good when you have a condition. For loops works well when you have an iterator. For example, in this case you use the variable `startFrequency` to go over the while loop until it reaches 108. With this information, how would you improve the code?